### PR TITLE
Add how to adjust httpd log level

### PR DESCRIPTION
**Pull request by jvlcek**
This PR adds instructions for bumping the httpd log level to debug.

### CODE REVIEW

**Comment by jvlcek** [comment]
@abellotti Can you please take a look.

File: site/_posts/2020-09-21-troubleshooting-containerized-oidc.md [hunk 1]
```diff
@@ -255,6 +256,55 @@ The following example shows how to check for the user's request headers from the
...
+  + Login
+
+  ```bash
+  $ oc login -u admin -p <pw> https://<my-container-host>/
+  ```
+
+  + Edit the httpd configmap:
+
+  ```bash
+  $ oc edit configmaps/httpd-configs -n joev-miq
```
Comment (abellotti) [inline site/_posts/2020-09-21-troubleshooting-containerized-oidc.md:274]: you probably want to be consistent with namespace with below, maybe replace with my-miq-project

File: site/_posts/2020-09-21-troubleshooting-containerized-oidc.md [hunk 2]
```diff
@@ -255,6 +256,55 @@ The following example shows how to check for the user's request headers from the
...
+
+  ```bash
+  $ oc get pods -n my-miq-project | grep httpd
+    httpd-abcdef1234-12345               1/1     Running   0          22d
+  ```
+
+  + Delete this httpd pod, which will result in a new one being created.
+
+  ```bash
+  $ oc delete pod httpd-abcdef1234-12345
```
Comment (abellotti) [inline site/_posts/2020-09-21-troubleshooting-containerized-oidc.md:292]: oc delete, oc logs, etc should all have -n my-miq-project unless you prepend these instructions with an oc project my-miq-project, then you can remove all the -n ...

Comment (jvlcek) [reply site/_posts/2020-09-21-troubleshooting-containerized-oidc.md]: Good catch! Thank you sir!

**Review by abellotti — APPROVED** [review]

**Comment by abellotti** [comment]
LGTM!! Thanks @jvlcek for this doc enhancement. 👍

**Comment by Fryguy** [comment]
Instead of adjusting blog posts can we eventually make an actual doc in manageiq-documentation?  People won't see this blog post update after some time, but documentation is forever.

**Pull request merged**